plutus-use-cases/test/Spec/Vesting.hs

```diff
@@ -152,7 +152,7 @@ startingBalance = Ada.adaValueOf 1000
 -- | Amount of money left in wallet `Wallet 2` after committing funds to the
 --   vesting scheme
 w2Funds :: Ledger.Value
-w2Funds = Value.minus startingBalance total
+w2Funds = 1 `timesFeeAdjustV` startingBalance `Value.minus` total
```

Comment:
Eeek - I have no idea how associativity of multiple backticked operators works. I think parens are a good idea here.